distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/CatalogBundle.java

```diff
@@ -168,12 +168,30 @@ public void setupCaching(boolean cachingEnabled) throws IOException {
         Hashtable<String, Object> updatedProperties = new Hashtable<>();
         updatedProperties.putAll(existingProperties);
         if (cachingEnabled) {
-            updatedProperties.put("cacheEnabled", "True");
+            updatedProperties.put("cacheEnabled", Boolean.TRUE);
         } else {
-            updatedProperties.put("cacheEnabled", "False");
+            updatedProperties.put("cacheEnabled", Boolean.FALSE);
         }
 
         Configuration configuration = adminConfig.getConfiguration(RESOURCE_DOWNLOAD_MANAGER_PID, null);
         configuration.update(updatedProperties);
     }
+
+    public boolean isCacheEnabled() throws Exception {
+        Map<String, Object> existingProperties = adminConfig.getDdfConfigAdmin()
+                .getProperties(RESOURCE_DOWNLOAD_MANAGER_PID);
+
+        if (existingProperties != null) {
+            if (existingProperties.get("cacheEnabled") instanceof Boolean) {
+                return (Boolean) existingProperties.get("cacheEnabled");
+            } else {
+                LOGGER.debug("cacheEnabled type is: {}",
+                        existingProperties.get("cacheEnabled").getClass());
```

Comment:
I think it should always be a boolean.  I have logging in there just in case someone changes it, we can see why it is failing.